app/src/endpoints.c

```diff
@@ -73,13 +80,17 @@ int zmk_endpoint_instance_to_str(struct zmk_endpoint_instance endpoint, char *st
     case ZMK_TRANSPORT_BLE:
         return snprintf(str, len, "BLE:%d", endpoint.ble.profile_index);
 
+    case ZMK_TRANSPORT_NONE:
+        return snprintf(str, len, "None");
+
     default:
         return snprintf(str, len, "Invalid");
     }
 }
 
 #define INSTANCE_INDEX_OFFSET_USB 0
 #define INSTANCE_INDEX_OFFSET_BLE ZMK_ENDPOINT_USB_COUNT
+#define INSTANCE_INDEX_OFFSET_NONE ZMK_ENDPOINT_USB_COUNT + ZMK_ENDPOINT_BLE_COUNT
```

Comment:
I think it makes more sense for "none" to be index 0. (Endpoints instances are explicitly documented as something that can change between firmware versions, so there's no need to maintain USB as 0)